### PR TITLE
feat(payment): PAYPAL-968 added banner widgets to page builder

### PR DESCRIPTION
**Pull request by andriiVitvitskyi1990**
#### What?
Added paypal banner widgets to page builder

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

-https://jira.bigcommerce.com/browse/PAYPAL-968

#### Screenshots (if appropriate)
Screenshot 2021-04-19 at 14 16 23

### CODE REVIEW

**Review by bc-fetisov — COMMENTED** [review]
Please see comments, and add explanation message to popup, see updated description https://jira.bigcommerce.com/browse/PAYPAL-968

Message example:
Screenshot 2021-04-14 at 18 15 08

File: schemaTranslations.json [hunk 1]
```diff
@@ -2047,5 +2047,74 @@
...
+    "default": "Product page banner"
+  },
+  "i18n.BannerStyleColor": {
+    "default": "Banner style color"
+  },
+  "i18n.White-no-border": {
+    "default": "White no border"
+  },
+  "i18n.gray": {
+    "default": "gray"
```
[the remark this answers is not in the document]
Comment (andriiVitvitskyi1990) [reply schemaTranslations.json]: fixed

File: schemaTranslations.json [hunk 2]
```diff
@@ -2047,5 +2047,74 @@
...
+  },
+  "i18n.monochrome": {
+    "default": "monochrome"
```
[the remark this answers is not in the document]
Comment (andriiVitvitskyi1990) [reply schemaTranslations.json]: fixed

File: schemaTranslations.json [hunk 3]
```diff
@@ -2047,5 +2047,74 @@
...
+  },
+  "i18n.grayscale": {
+    "default": "grayscale"
```
[the remark this answers is not in the document]
Comment (andriiVitvitskyi1990) [reply schemaTranslations.json]: fixed

File: config.json [hunk 4]
```diff
@@ -331,6 +331,14 @@
     "paymentbuttons-paypal-label": "checkout",
     "paymentbuttons-paypal-tagline": false,
     "paymentbuttons-paypal-fundingicons": false,
+    "paymentbanners-homepage-color": "white",
+    "paymentbanners-homepage-ratio": "8x1",
+    "paymentbanners-homepage-logo-type": "primary",
```
[the remark this answers is not in the document]
Comment (andriiVitvitskyi1990) [reply config.json]: fixed

File: schema.json [hunk 5]
```diff
@@ -3062,5 +3062,211 @@
...
+        "id": "paymentbanners-cartpage-logo-type",
+        "force_reload": true,
+        "options": [
+          {
+            "value": "inline",
+            "label": "i18n.inline"
+          },
+          {
+            "value": "primary",
+            "label": "i18n.right"
```
[the remark this answers is not in the document]
Comment (andriiVitvitskyi1990) [reply schema.json]: added

File: schemaTranslations.json [hunk 6]
```diff
@@ -2047,5 +2047,80 @@
...
+    "default": "Alternative"
+  },
+  "i18n.none": {
+    "default": "None"
+  },
+  "i18n.PaymentBanners": {
+    "default": "Payment Banners"
+  },
+  "i18n.CartPageBanner": {
+    "default": "Cart Page Banner"
```
Comment (bc-fetisov) [inline schemaTranslations.json:2118]: change letter ->`Cart page Banner`

File: schemaTranslations.json [hunk 7]
```diff
@@ -2047,5 +2047,80 @@
...
+    "default": "Inline"
+  },
+  "i18n.alternative": {
+    "default": "Alternative"
+  },
+  "i18n.none": {
+    "default": "None"
+  },
+  "i18n.PaymentBanners": {
+    "default": "Payment banners"
```
Comment (bc-fetisov) [inline schemaTranslations.json:2115]: `Payment banners` > `Payment Banners`

File: schema.json [hunk 8]
```diff
@@ -3062,5 +3062,215 @@
...
+        "label": "i18n.BannerStyleLogoType",
+        "id": "paymentbanners-cartpage-logo-type",
+        "force_reload": true,
+        "options": [
+          {
+            "value": "inline",
+            "label": "i18n.inline"
+          },
+          {
+            "value": "i18n.primary",
```
Comment (bc-fetisov) [inline schema.json:3261]: `i18n.primary` > `primary`

Comment (andriiVitvitskyi1990) [reply schema.json]: fixed

File: schemaTranslations.json (hunk 7 above)
Comment (andriiVitvitskyi1990) [reply schemaTranslations.json]: fixed

**Review by BC-ACherednichenko — APPROVED** [review]

**Review by bc-fetisov — APPROVED** [review]
LGTM. Please update change log only.
@BC-tymurbiedukhin @bc-alexsaiannyi could you approve this PR also?

**Comment by andriiVitvitskyi1990** [comment]
> LGTM. Please update change log only.
Updated change log

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

**Review by bc-alexsaiannyi — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]

**Pull request merged**